server/routerlicious/packages/test-utils/src/testDocumentStorage.ts

```diff
@@ -191,6 +192,7 @@ export class TestDocumentStorage implements IDocumentStorage {
                 scribe: undefined,
                 tenantId,
                 version: "0.1",
+                lastSummarizedSequenceNumber: -1,
```

Comment:
This is a scribe property so this should be inside scribe object. Not a top level one.